pkg/client/didexchange/client.go

```diff
@@ -4,6 +4,21 @@ Copyright SecureKey Technologies Inc. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
+// Package didexchange enables relationship between two agents via DID Exchange Protocol.
```

Comment:
I feel it would be better to have package comments in pkg level doc.go file. Here is a [reference](https://github.com/golang/go/blob/master/src/fmt/doc.go).

Reply:
Done